contracts/pcv/PCVGuardian.sol

```diff
@@ -0,0 +1,181 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {EnumerableSet} from "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+import {CoreRef} from "../refs/CoreRef.sol";
+import {IPCVGuardian} from "./IPCVGuardian.sol";
+import {IPCVDeposit} from "./IPCVDeposit.sol";
+import {CoreRefPauseableLib} from "../libs/CoreRefPauseableLib.sol";
+import {TribeRoles} from "../core/TribeRoles.sol";
+
+/// @notice PCV Guardian is a contract to safeguard protocol funds
+/// by being able to withdraw whitelisted PCV deposits to an immutable safe address
+contract PCVGuardian is IPCVGuardian, CoreRef {
+    using CoreRefPauseableLib for address;
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    ///@notice set of whitelisted pcvDeposit addresses for withdrawal
+    EnumerableSet.AddressSet private whitelistAddresses;
+
+    ///@notice safe address where PCV funds can be withdrawn to
+    address public immutable safeAddress;
```

Comment:
Is this flexible enough? Currently you'll only ever be able to withdraw funds to one address through this contract. Do you not need any more redundancy in case of emergency?

What is the rationale for restricting yourselves to one safe address?

Reply:
Okay. In that case, I would personally lean towards giving yourself more flexibility by adding a `setSafeAddress() onlyGovernor` method. 

You don't ever need to use, but it would give you some breathing room to change the safe address if there's ever a need to

Reply:
If there is ever a need to update the safe address, this contract will be redeployed

Reply:
IMO easier to have an onlyGov method to change it. Up to you though.

Reply:
yes it's easier to be able to change it from an upgrade perspective, but during an emergency situation, you need clarity and low cognitive overhead when thinking through the effects of decisions. giving optionality for where PCV can be moved during an emergency is not optimal as it opens too many decision trees